commerce_bluesnap.module

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Provides Commerce integration with the BlueSnap Payment Gateway.
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_help().
+ */
+function commerce_bluesnap_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    case 'help.page.commerce_bluesnap':
+      $output = '';
+      $output .= '<h2>' . t('About') . '</h2>';
+      $output .= '<p>' . t('Provides Drupal Commerce integration with the BlueSnap Payment Platform.') . '</p>';
+      $output .= '<h3>' . t('Features') . '</h3>';
+      $output .= '<ul>';
+      $output .= '<li>' . t(
+          '<strong>Credit Card Payments Integration</strong>
+          - Hosted Payment Fields payment gateway. Serves payment input fields
+          from Bluesnap’s domain, minimizing the scope of PCI compliance.
+          <br> For more info, see the README.txt file.
+          Configure <a href=":configuration_link">here</a>.', [
+            ':configuration_link' => Url::fromUri(
+              'internal:/admin/commerce/config/payment-gateways/manage/bluesnap'
+            )->toString()
+          ]
+        ) .
```

Comment:
**OPTIONAL CHANGE REQUEST**

*Readability*

Even though the following is pretty common

```
some_function($arg1, [
  'key' => 'value',
]);
```
it's not ideal and not as readable because you're mixing the arguments with the function name. The following is better:
```
some_function(
  $arg_1,
  ['key' => 'value']
);
```

In real code the 2nd makes it much easier, for example in this particular situation to easily spot where the string (1st argument) ends and the 2nd argument begins.

```
$output .= '<li>' . t(
  '<strong>Credit Card Payments Integration</strong>
  - Hosted Payment Fields payment gateway. Serves payment input fields
  from Bluesnap’s domain, minimizing the scope of PCI compliance.
  <br> For more info, see the README.txt file.
  Configure <a href=":configuration_link">here</a>.',
  [
    ':configuration_link' => Url::fromUri(
      'internal:/admin/commerce/config/payment-gateways/manage/bluesnap'
    )->toString()
  ]
) .
```

This is optional, you can keep it as is if you prefer.